packages/caliper-publish/caliper.Dockerfile

```diff
@@ -12,7 +12,7 @@
 # limitations under the License.
 #
 
-FROM node:10.22-alpine
+FROM node:14.19-alpine
```

Comment:
This change breaks the docker build so recommend for this PR that you revert it.